oauthenticator/tests/test_globus.py

```diff
@@ -54,6 +54,22 @@ def globus_client(client):
     return client
 
 
+@fixture
+def mock_globus_user(mock_globus_sdk):
+    class User:
+        name = 'Wash'
+        state = {'tokens': mock_globus_sdk.by_resource_server}
+
+        @gen.coroutine
+        def get_auth_state(self):
+            return self.state
+
+        @gen.coroutine
+        def save_auth_state(self, state):
```

Comment:
Can this perhaps be async instead of @gen.coroutine functions?

Reply:
Absolutely, I'll update that now.